client/src/components/navbar/Navbar.css

```diff
@@ -88,15 +88,15 @@
 
     .navbar-logo {
         position: absolute;
-        top: 0;
+        top: -10px;
```

Comment:
Look into flex-box. Using "position: absolute" is a sin!

Reply:
gg!